kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js

```diff
@@ -367,7 +362,7 @@ define([
 
             bus.on('toggle-advanced', function () {
                 settings.showAdvanced = !settings.showAdvanced;
-                renderAdvanced();
+                showHideAdvanced();
```

Comment:
updated so that we can just show/hide as needed, rather than re-rendering everything